src/ng-select/lib/ng-select.component.ts

```diff
@@ -168,6 +168,7 @@ export class NgSelectComponent implements OnDestroy, OnChanges, OnInit, AfterVie
 	element: HTMLElement;
 	focused: boolean;
 	escapeHTML = true;
+	finalTabFocus = true;
```

Comment:
Delete this and use `_mergeGlobalConfig` method to merge both

Reply:
I think I need another variable here, and also have to call the update on ngOnChanges since the property might be different for each dropdown. If I make it one value that gets set during the mergeGlobalConfig, it can never get reset when you move to the next dropdown

Reply:
Follow [similar pattern](https://github.com/ng-select/ng-select/blob/master/src/ng-select/lib/ng-select.component.ts#L1048). Define private variable like `_tabFocusOnClear` and store derived value in that.

Reply:
Is this ok now? I changed it to a private variable but I did also need to add a getter so that I can fetch the value in the html. Let me know if you think there's a better approach, thanks!